app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+    "authorities_removal_alert_proceed": "Continuar",
+    "authorities_removal_alert_title": "Remover autoridade",
+    "authorities_title": "Fontes Confiáveis",
+    "choose_provider_subtitle": "Para ser informado de exposição você deverá contatar uma agência de saúde.",
+    "choose_provider_title": "Escolha a autoridade de saúde",
+    "commitment": "Comprometimento",
+    "commitment_para": "Safe Paths armazena e checa sua interação com as pessoas usando sua localização. Os seus dados NUNCA sairão dos seu celular sem o seu consentimento.",
+    "default_news_site_name": "Novidades do Safe Paths",
+    "event_history_subtitle": "Entender sua exposição pessoal baseada na informação compartilhada com as autoridades de saúde.",
+    "event_history_title": "Histórico de exposição",
+    "export_para_1": "Se o seu teste para o COVID-19 deu positivo, por favor faça sua parte compartilhando o seu histórico de localização para as autoridades de saúde locais.",
+    "export_para_2": "A sua localização é compartilhada como uma lista simples de horários e locais, sem nenhuma informação além disso.",
+    "home_at_risk_header": "Você pode ter sido exposto",
+    "home_at_risk_subsubtext": "Isso não significa que você está infectado.",
+    "home_at_risk_subtext": "Baseado em seu histórico de localização, é possível que você esteve em contato ou próximo a alguém diagnosticado com COVID-19.",
+    "home_enable_location": "Ativar dados de localização",
+    "home_mayo_link_heading": "Mais informações sobre o COVID-19",
+    "home_mayo_link_label": "de Mayo Clinic",
+    "home_no_contact_header": "Nenhum contato conhecido",
+    "home_no_contact_subtext": "Com base nos dados disponíveis, você não esteve perto de ninguém que testou positivo para o COVID-19.",
+    "home_unknown_header": "Desconhecido",
+    "home_unknown_subtext": "Não podemos dizer se você está em risco, a menos que você permita o aplicativo acessar sua localização.",
+    "home_setting_off_header": "Desconhecido",
+    "home_setting_off_subtext": "Não podemos dizer se você está em risco, a menos que você ative seu histórico de localização na tela de configurações.",
+    "import_step_1": "1. Realize login na sua conta do Google e baixe o seu histórico de localização",
+    "import_step_2": "2. Depois que você baixar, abra esta tela novamente. Os dados serão importados automaticamente.",
+    "import_title": "Importar localização",
+    "latest_news": "Últimas notícias",
+    "launch_done_header": "Tudo pronto",
+    "launch_done_subheader": "Você está pronto para seguir. Lembre-se, você sempre pode alterar suas preferências depois. ",
+    "launch_enable_location": "Ativar localização",
+    "launch_enable_notif": "Ativar notificações",
+    "launch_finish_set_up": "Concluir configurações",
+    "launch_get_started": "Iniciar",
+    "launch_location_access": "Acesso à localização",
+    "launch_location_header": "Para lembrar onde você deve ir, seu celular precisa ter sua localização",
```

Comment:
The English line is:
To remember where you go, your phone needs to save your location.
Does this translate correctly here?

Should it be: Para lembrar onde você vai?

Reply:
Oh, sorry. I will correct it today!